src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -10,11 +10,13 @@
             </div>
             <div *ngFor='let comment of workItem.relationalData.comments; let counter = index' class="comments-wrap">  
                 <div>
-                    <img id="{{'comment_avatar_' + counter}}"
-                      class="user-assign-avatar pull-left" 
-                      *ngIf="comment.relationalData.creator?.attributes?.imageURL" 
-                      [src]="comment.relationalData.creator?.attributes?.imageURL | almAvatarSize:30"
-                    />
+                    <div class="user-avatar pull-left">
+                        <img id="{{'comment_avatar_' + counter}}"
+                        class="user-assign-avatar pull-left" 
```

Comment:
Please indent these 3 lines under the img tag.
Also, indention should be 2 spaces.